projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/graphql/model/DataJobExecutionOrder.java

```diff
@@ -30,7 +30,8 @@
          DataJobExecution_.LAST_DEPLOYED_BY,
          DataJobExecution_.STARTED_BY,
          DataJobExecution_.STATUS,
-         DataJobExecution_.VDK_VERSION);
+         DataJobExecution_.VDK_VERSION,
+         DataJobExecution_.DATA_JOB + "." + DataJob_.NAME);
```

Comment:
As you can see from GraphQL schema (DataJobExecution) the public field is `jobName` not the `dataJob.name`. Probably we need mapping between public objects and db entities.